courseraprogramming/commands/sanity.py

```diff
@@ -40,10 +40,7 @@ def command_sanity(args):
         docker_version = conn.version()
         logging.info("Docker version: %s", docker_version["Version"])
         if semver.match(docker_version["Version"], '>1.5.0'):
```

Comment:
What about support for 1.11. Some instructors have already started using it. Have we tested it out yet, if not, we might want to add that condition here?